src/eip4824.ts

```diff
@@ -1,9 +1,9 @@
 import express from 'express';
-import { getSpace } from './helpers/spaces';
+import { getCombinedMembersAndVoters, getSpace } from './helpers/spaces';
 import db, { sequencerDB } from './helpers/mysql';
 
 const router = express.Router();
-const context = '<http://www.daostar.org/schemas>';
+const context = ["https://www.snapshot.org", "https://www.daostar.org/schemas"];
```

Comment:
We don't use `www` on snapshot.org, also we use single quote, if you run the lint script it should fix that
```suggestion
const context = ['https://snapshot.org', 'https://www.daostar.org/schemas'];
```